wgpu-core/Cargo.toml

```diff
@@ -16,10 +16,11 @@ license = "MPL-2.0"
 
 [features]
 default = []
-trace = ["ron", "serde", "wgt/trace"]
-replay = ["serde", "wgt/replay"]
 #NOTE: glutin feature is not stable, use at your own risk
 #glutin = ["gfx-backend-gl/glutin"]
+profiling = ["tracing"]
```

Comment:
We talked about this on the Matrix, and I think we should just unconditionally switch from `log` to `tracing`. Not making it optional. The *subscribers* will be optional wherever we hook them up.